hang_man.py

```diff
@@ -0,0 +1,158 @@
+import os
+import sys
+import copy
+import string
+import random
+import argparse
+import urllib.request
+
+
+class Hangman:
+
+    def __init__(self, min_limit):  # Creating init method that takes in max word limit as perimeter
+        self.min_limit = min_limit  # Minimum word limit taken from user
+        self.word_list = []  # A list that stores all the word from local file or web
+        self.right_list = []  # A list that stores the right word
+        self.guess_letter = str  # User input of their guessed character
+        self.hang_list = [       # A list that contains all the stages of hangman
+            """
+            |‾‾‾‾‾‾‾‾|
+            |       
+            |      
+            |      
+            |
+            """,
+            """
+            |‾‾‾‾‾‾‾|
+            |       O
+            |   
+            |   
+            |
+            """,
+            """
+            |‾‾‾‾‾‾‾|
+            |       O
+            |      /
+            |   
+            |
+            """,
+            """
+            |‾‾‾‾‾‾‾|
+            |       O
+            |      /|
+            |   
+            |
+            """,
+            """
+            |‾‾‾‾‾‾‾|
+            |       O
+            |      /|\\
+            |   
+            |
+            """,
+            """
+            |‾‾‾‾‾‾‾|
+            |       O
+            |      /|\\
+            |      / 
+            |
+            """,
+            """
+            |‾‾‾‾‾‾‾|
+            |       O
+            |      /|\\
+            |      / \\
+            |
+            """]
+
+    def access_doc(self):
+        url = 'http://norvig.com/ngrams/sowpods.txt'  # Create a variable that stores for URL for text file
+        file_exist = os.path.exists('./word_data/words.txt')  # Boolean value to check if text file exist first
```

Comment:
You can create a file path variable to dry up the code in the access_doc method

Reply:
Added a new variable under that method called file_loc